specification/monitor/resource-manager/Microsoft.Insights/preview/2017-11-01-preview/baseline_API.json

```diff
@@ -41,7 +41,7 @@
           "tags": [
             "Baseline"
           ],
-          "operationId": "MetricBaseline_Get",
+          "operationId": "Baseline_Get",
```

Comment:
operationId updates will cause breaking changes in existing SDKs, are you intentionally making this change? The operationIds in the specification correspond to the name and grouping of the methods when generating SDK code. 
See https://github.com/Azure/azure-sdk-for-python/pull/4714/files#diff-ca529633360fbb2a6253fb399e08cf29R18 as an example

Reply:
It caused a conflict in the build of the Java SDK, but I reverted the breaking change and updated the new API